src/symbols/Label.tsx

```diff
@@ -68,10 +104,70 @@ export const Label: FC<LabelProps> = ({
     () => (stroke ? new Color(stroke) : undefined),
     [stroke]
   );
+  const normalizedBackgroundColor = useMemo(
+    () => (backgroundColor ? new Color(backgroundColor) : null),
+    [backgroundColor]
+  );
+  const normalizedStrokeColor = useMemo(
+    () => (strokeColor ? new Color(strokeColor) : null),
+    [strokeColor]
+  );
+  // Normalize the radius to be between 0 and 3
+  const normalizedRadius = useMemo(
```

Comment:
I'm wondering if this memo costs more than its saving since its just simple math.